sky/templates/aws-ray.yml.j2

```diff
@@ -96,12 +99,15 @@ setup_commands:
     (pip3 list | grep ray | grep {{ray_version}} 2>&1 > /dev/null || pip3 install -U ray[default]=={{ray_version}}) && mkdir -p ~/sky_workdir && mkdir -p ~/.sky/sky_app;
     pip3 uninstall skypilot -y &> /dev/null;
     pip3 install "$(echo {{sky_remote_path}}/skypilot-{{sky_version}}*.whl)[aws]";
-    python3 -c "from sky.skylet.ray_patches import patch; patch()";
     sudo systemctl stop unattended-upgrades;
     sudo kill -9 `sudo lsof /var/lib/dpkg/lock-frontend | awk '{print $2}' | tail -n 1` || true;
     sudo pkill -9 apt-get;
     sudo pkill -9 dpkg;
     sudo dpkg --configure -a;
+    sudo bash -c 'rm -rf /etc/security/limits.d; echo "* soft nofile 1048576" >> /etc/security/limits.conf; echo "* hard nofile 1048576" >> /etc/security/limits.conf';
+    sudo grep -e '^DefaultTasksMax' /etc/systemd/system.conf || (sudo bash -c 'echo "DefaultTasksMax=infinity" >> /etc/systemd/system.conf'); sudo systemctl set-property user-$(id -u $(whoami)).slice TasksMax=infinity; sudo systemctl daemon-reload;
```

Comment:
Is this line required? Not very common to see this set.

Reply:
Yes, the previous behavior we observed in the description of #1125 is that ray spawns too many thread > 10813 limit for the user by default, causing the raylet died due to the `fail to fork` system error. By monitoring the number of threads with `watch -n1 “ps -fLu gcpuser | wc -l”`, the ray job submit will stuck after the number goes beyond the limit, before this line was added, but it will successfully pass the number and keep submitting after this line added.

Reply:
Deserve a comment?

Reply:
Added. Thanks!